demos/icon-toggle.html

```diff
@@ -24,11 +24,6 @@
         background: #303030;
       }
 
-      .mdc-theme--dark:not(.demo-color-combo) {
```

Comment:
If you're going to remove this then I'd make sure the non-disabled dark theme example still has enough padding such that the entire ripple can be contained within the dark background.